packages/rspack/etc/core.api.md

```diff
@@ -1967,8 +1965,8 @@ export const experiments: Experiments_2;
 interface Experiments_2 {
     // (undocumented)
     globalTrace: {
-        register: typeof registerGlobalTrace;
-        cleanup: typeof cleanupGlobalTrace;
+        register: (filter: string, layer: "chrome" | "logger" | "otel", output: string) => Promise<void>;
```

Comment:
This is a breaking change to the register API but since it's experimental, so I guess it's fine.